src/index.html

```diff
@@ -1,13 +1,343 @@
+                  <img
+                    src="/images/Ear-box.png"
+                    alt="All Products"
+                    class="category__photo"
+                  >
+                </a>
+              </div>
+              <h2 class="category__title">All Products</h2>
+            </article>
+        
+            <article class="category">
+              <div class="category__photos">
+                <a href="#" class="category__link category__link--square">
+                  <img
+                    src="/images/Ear.png"
+                    alt="Audio"
+                    class="category__photo"
+                  >
+                </a>
+                <a href="#" class="category__link category__link--wide">
+                  <img
+                    src="/images/Ear-stick.png"
+                    alt="Audio"
+                    class="category__photo"
+                  >
+                </a>
+              </div>
+              <h2 class="category__title">Audio</h2>
+            </article>
+            <article class="category">
+              <div class="category__photos">
+                <a href="#" class="category__link category__link--wide">
+                  <img
+                    src="/images/Black-phone.png"
+                    alt="Black phone"
+                    class="category__photo"
+                  >
+                </a>
+                <a href="#" class="category__link category__link--square">
+                  <img
+                    src="/images/Charger.png"
+                    alt="Accessories"
+                    class="category__photo"
+                  >
+                </a>
+              </div>
+              <h2 class="category__title">Accessories</h2>
+            </article>
+            <a href="#categories" class="view-all">View all</a>
+          </div>
+        </section>
+      </div>
+
+      <div class="division-background"></div>
+
+      <section class="about-us" id="about-us">
+        <div class="container">
+          <div class="about-us__container">
+            <h2 class="section-title about-us__title">About us</h2>
+            <div class="about-us__text-container">
+              <p class="about-us__text">
+                We&#39;re a London-based tech company on a mission to remove barriers between people and technology.<br><br>
+                To do this, we&#39;re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+                No confusing tech-speak. No silly product names.Just artistry, passion and trust.
+                And products we&#39;re proud to share with our friends and family. Simple.<br><br>
+                Because beautiful tech shouldn&#39;t be complicated.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+    </main>
+    <div class="container">
+      <footer class="footer" id="footer">
```

Comment:
For better semantics, it is worth making the `footer` the main parent tag for the lower block, not a `div`.